python/cugraph/tests/test_pagerank.py

```diff
@@ -159,16 +158,16 @@ def test_pagerank(managed, pool, graph_file, max_iter, tol, alpha,
     rmm.initialize()
 
     assert(rmm.is_initialized())
-    M = utils.read_mtx_file(graph_file+'.mtx')
+    M = utils.read_csv_for_nx(graph_file)
     networkx_pr, networkx_prsn = networkx_call(M, max_iter, tol, alpha,
                                                personalization_perc)
 
     cu_nstart = None
     if has_guess == 1:
         cu_nstart = cudify(networkx_pr)
-        max_iter = 5
+        max_iter = 500
```

Comment:
500 seems like a lot of iterations.  I thought 50 or 100 was more common

Reply:
That's residue from trying to get it to work with the directed data set.  I didn't find a good number, but 50 wasn't sufficient.  500 converged but got different answers between networkx and cugraph for some combinations.  But I turned the directed data sets off so I'll set it back to 5.

A successful call exits once we converge to the tolerance, so most of the runs were just as fast as long as max_iter was large enough.